src/views/Profile/AuthorsList/styles/AuthorsList.styles.ts

```diff
@@ -0,0 +1,30 @@
+import { makeStyles, Theme } from '@material-ui/core';
+
+export const useStyles = makeStyles((theme: Theme) => ({
+  mainButton: {
+    padding: theme.spacing(2, 3),
+    color: '#ffffff',
+    backgroundColor: '#73DDFF',
```

Comment:
may those colors be taken from a theme object?

Reply:
may we want add it into theme object then? what is semantic meaning of that color? "normal button color"? "special ability"? anything else?

Reply:
As far as I can see this color is used 3 times for 3 different components, also there are other components that use similar color but different shades. I'm not sure if it was done on purpose or this is design flaw.
I'll see what we can do.